packages/serverless/src/gcpfunction.ts

```diff
@@ -0,0 +1,115 @@
+import { HttpFunction } from '@google-cloud/functions-framework/build/src/functions';
```

Comment:
Is this import magically available on GCP?

Reply:
It doesn't really matter cause it's a type-only import which is erased in the final build.

I wanted to explicitly assert that we only import type definitions but current TypeScript version doesn't yet support `import type` syntax.
I'll leave a TODO-comment for this both to clarify this uncertainy & to easily face the issue in the future when TypeScript is upgraded. As I see It's not easy to bump the TypeScript version in the project right now because further versions after `3.7.5` contain some breaking changes so the TS upgrade can only be done in major release (see #2848).